### ISSUE TITLE
> 140 character tweets not working via extension

### ISSUE THREAD

**Issue by qdot** [quinncomendant/280Characters#2]
I'm pretty sure twitter is shutting down usage of weighted_character_count via the web client API. I've tested it in both firefox using manual post query changes and chrome using your extensions, and while it was working around 11am today, it no longer seems to work now. This is most likely being rolled out to servers slowly, so it may depend on which endpoint you're hitting for the next bit, but I would expect things to be dead by the end of the day.

Building 280 character tweets via queries using the mobile app keys still seems to work.

**Comment by quinncomendant** [comment]
Yes, this seems to be the case. RIP weighted_character_count on twitter.com. ⚰️

Since tweetdeck is still working, one option is to override the twitter.com's `wVpQ.s.sendTweet()` method to post to `https://api.twitter.com/1.1/statuses/update.json` instead of `https://twitter.com/i/tweet/create`, which would also require generating a valid `x-csrf-token`, authentication headers, and anything else required by that endpoint. I'm just going to leave this idea here in case anyone wants to pick it up. ;)
